product_docs/docs/migration_portal/4/known_issues_notes.mdx

```diff
@@ -243,6 +245,59 @@ While using the Oracle default case, you may experience a lower compatibility ra
 -   The EDB DDL Extractor currently doesn't support the extraction of `ROLES`, `SYSTEM GRANTS ON ROLES`, `OBJECT GRANTS ON ROLES`, and `ROLE GRANTS` from Oracle 11g. This behavior results in error messages being written to the extracted files in the sections corresponding to these object types. These errors don't cause any issue in the assessment of these files by Migration Portal.
 -   The EDB DDL Extractor script may log `object "OBJECT_NAME" of type SYNONYM not found in schema "PUBLIC"` errors in the dependent objects section of the extracted file. This happens only if the user selects the option to extract dependent objects from an Oracle multi tenant environment where the Oracle database is a container database.
 
+### "Snapshot Too Old" Error
+
+The EDB DDL Extractor displays the error “ORA-01555: snapshot too old“ at runtime when the database server generates a volume of transactions that cannot be properly processed by the UNDO tablespace.
+
+When the database server generates undo transactions at a high rate, it can cause the server to run out of space to store undo data. Since the UNDO tablespace is implemented as a circular buffer, it starts overwriting older undo data blocks. This error allows the migration to continue, but may cause it to skip migrating public synonym database objects.
+
+To work around this error, increase the allocated space for the UNDO_RETENTION parameter.
+
+First, check the currently allocated total space for undo operations and the currently free space:
+
+1.  Check the existing value of the UNDO_RETENTION parameter:
+
+    ```sql
+    SELECT value FROM v$parameter WHERE name = 'undo_retention';
+    ```
+
+1.  Add the results of the two following queries to check the total available space in the UNDO tablespace:
+
+    ```sql
+    SELECT SUM(bytes)/1024/1024 free_mb FROM dba_undo_extents WHERE status='EXPIRED';
+    ```
+
+    ```sql
+    SELECT dfs.tablespace_name, SUM(dfs.bytes) / 1024 / 1024 AS free_mb FROM 
+    dba_free_space dfs 
+    JOIN dba_tablespaces dt ON dfs.tablespace_name = dt.tablespace_name 
+    WHERE dt.contents = 'UNDO' GROUP BY dfs.tablespace_name; 2 3 4
+    ```
+
+Next, determine the number of MB your environment required and increase the allocated storage space:
+
+1.  Monitor the volume of undo data generated by transactions:
+    
+    ```sql
+    SELECT SUM(undoblks * 8192)/1024/1024 AS undo_mbs FROM v$undostat;
+    ```
+
+    !!!note 
+    The resulting value reflects the used MB over a time interval specified in your system's `v$undostat` view.
```

Comment:
@gabbasb  I added this note to have some sort of guidance for the customer to understand the impact of the value. I wonder is we want to advise them to look up the time interval with something like `SELECT MIN(begin_time), MAX(end_time) FROM v$undostat;` or something similar?

Reply:
I don't think so.